parser/sql/dialect/doris/src/main/antlr4/imports/doris/DDLStatement.g4

```diff
@@ -359,6 +359,43 @@ createView
       (WITH (CASCADED | LOCAL)? CHECK OPTION)?
     ;
 
+// DORIS ADDED BEGIN
+createMaterializedView
+    : CREATE MATERIALIZED VIEW ifNotExists? name
```

Comment:
Please keep the systax in one line unless it contains more than 5 branches. You can read SQL Parser of Conduct - https://shardingsphere.apache.org/community/en/involved/conduct/code/#sql-parser-of-conduct